repoze/lru/__version__.py

```diff
@@ -0,0 +1,2 @@
+version = (0, 7, 1)
+str_version = '.'.join(['{0}'.format(v) for v in version])
```

Comment:
Please kill this file, and the indirection below in `setup.py`:  the right way to find out what version of a package is installed is to use `pkg_resources`.